### ISSUE TITLE
Printing data instance for which the prediction is made in simple txt format.

### ISSUE THREAD

**Issue by srijanc** [marcotcr/lime#410]
I am unable to get the data instance output in a simple format for which the prediction is done.
I see that the html file generated by save_to_file contains the desired output.

```
exp_div = top_div.append('div').classed('lime explanation', true);
--
  | exp.show([["Capital Gain <= 0.00", -0.6988713302192259], ["Age <= 28.00", -0.10341048820226414], ["Hours per week <= 40.00", -0.09778069895349602], ["Marital Status=Never-married", -0.09644688889745717], ["Education-Num <= 9.00", -0.09460883490060283], ["Occupation=Other-service", -0.03812212203159269], ["Capital Loss <= 0.00", -0.03757078697144384], ["Sex=Male", 0.03494924726046984], ["Relationship=Other-relative", -0.024712102230666267], ["Race=White", 0.020325648985441164], ["Workclass=Private", 0.007045297401002338], ["Country=United-States", 0.006050764450323902], ["Education=11th", -0.003516358172806853], ["178587.00 < fnlwgt <= 237549.25", 0.0031683848402579524]], 1, exp_div);

```
Is there a way to get this specific output ?

**Comment by marcotcr** [comment]
`exp.as_list()`

**Comment by srijanc** [comment]
exp.as_list() gives me just the prediction output but not the data instance on which the prediction is made (that is what I am seeking). Please correct me if wrong.

**Comment by marcotcr** [comment]
I am not sure what you mean by data instance. `exp.as_list()` gives you precisely the output out pasted above, which is the explanation feature names with respective weights **for the instance being explained**

**Comment by srijanc** [comment]
Oh.....sorry it seems that I have posted a wrong screenshot. Yes, you are correct.
![image](https://user-images.githubusercontent.com/43124192/71386044-2b54e300-263f-11ea-9040-1100fc9ad8b4.png)

So, in this screenshot if you see, the left hand side is the explanation along with its weight and on right hand side, it is the data instance on which the explanation is generated. **I am looking for that data instance which is the test instance/input instance on which the explanation is generated.** I have extracted the data from the HTML output generated using save_to_file().
